src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -6,3 +6,96 @@
 from azure.cli.help_files import helps #pylint: disable=unused-import
 
 #pylint: disable=line-too-long
+
+helps['network dns record-set create'] = """
```

Comment:
That makes sense to me.

---
In reply to: [75767560](https://github.com/Azure/azure-cli/pull/718#discussion_r75767560) [](ancestors = 75767560,75752065,75560645,75556265)